### ISSUE TITLE
Can't get remote versions

### ISSUE THREAD

**Issue by tskarhed** [olsonpm/lilyvm#1]
Hello!

I just installed this on Ubuntu 22.04.2 LTS, and it seems like it fails to fetch the available versions.

```
lilyvm install -v 2.24.1

Error: Not installing - The version '2.24.1' is not in the list
  of available versions.  The list can be shown via 'lilyvm ls-remote --show-all'.
```

```
lilyvm ls-remote --show-all

* = currently installed

```

**Comment by olsonpm** [comment]
thanks for letting me know.  I'll have time to look into it this week.  Probably the endpoint i used has changed in the past 8 years

**Comment by olsonpm** [comment]
@tskarhed alright this should be fixed as of [e3c6009](https://github.com/olsonpm/lilyvm/commit/e3c600912826cede45a426cb438d3e5b5fb2cd03?w=1).  You should be able to update via `lilyvm update` and be on your way
